tests/api_connexion/endpoints/test_import_error_endpoint.py

```diff
@@ -0,0 +1,153 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import unittest
+
+from parameterized import parameterized
+
+from airflow.models.errors import ImportError  # pylint: disable=redefined-builtin
+from airflow.utils import timezone
+from airflow.utils.session import provide_session
+from airflow.www import app
+from tests.test_utils.db import clear_db_import_errors
+
+
+class TestBaseImportError(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls) -> None:
+        super().setUpClass()
+        cls.app = app.create_app(testing=True)  # type:ignore
+
+    def setUp(self) -> None:
+        self.client = self.app.test_client()  # type:ignore
+        self.timestamp = "2020-06-10T12:02:44.870706"
```

Comment:
Can you reduce the precision of the dates given? Now it is difficult to compare them.